cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -473,8 +475,8 @@ _sendEKGDirectInt ekgDirect name val = do
         Gauge.set gauge (fromIntegral val)
         pure $ SMap.insert name gauge registeredMap
 
-_sendEKGDirectDouble :: EKGDirect -> Text -> Double -> IO ()
-_sendEKGDirectDouble ekgDirect name val = do
+sendEKGDirectDouble :: EKGDirect -> Text -> Double -> IO ()
+sendEKGDirectDouble ekgDirect name val = do
```

Comment:
The inner function returns with `pure ...` which is then ignored by `modifyMVar_`. Even though it's outside of the scope, could you fix it?

Reply:
I'm not addressing feedback related to  `sendEKGDirectDouble` , `sendEKGDirectCounter`, and `_sendEKGDirectInt` . This PR is for block delay counters, not general tracing improvements.

Reply:
Totally fair -- and @coot, I'd like to remind that that code is being retired quite soon -- and, we can speed it up by helping with reviews on relevant PRs.. : -)